src/test/unit/common/languageServer/languageServer.test.ts

```diff
@@ -2,26 +2,25 @@
 import assert, { deepStrictEqual } from 'assert';
 import { ReplaySubject } from 'rxjs';
 import sinon from 'sinon';
+import { v4 } from 'uuid';
 import { IAuthenticationService } from '../../../../snyk/base/services/authenticationService';
 import { IConfiguration } from '../../../../snyk/common/configuration/configuration';
 import { LanguageServer } from '../../../../snyk/common/languageServer/languageServer';
 import { InitializationOptions } from '../../../../snyk/common/languageServer/settings';
-import { IContextService } from '../../../../snyk/common/services/contextService';
 import { DownloadService } from '../../../../snyk/common/services/downloadService';
+import { User } from '../../../../snyk/common/user';
 import { ILanguageClientAdapter } from '../../../../snyk/common/vscode/languageClient';
 import { LanguageClient, LanguageClientOptions, ServerOptions } from '../../../../snyk/common/vscode/types';
 import { IVSCodeWorkspace } from '../../../../snyk/common/vscode/workspace';
-import { extensionContextMock } from '../../mocks/extensionContext.mock';
 import { LoggerMock } from '../../mocks/logger.mock';
 import { windowMock } from '../../mocks/window.mock';
 import { stubWorkspaceConfiguration } from '../../mocks/workspace.mock';
 
 suite('Language Server', () => {
-  const extensionContext = extensionContextMock;
   const authService = {} as IAuthenticationService;
+  const user = new User(v4(), undefined);
```

Comment:
why not v1?

Reply:
current logic used v4 for that. But it doesn't matter as both are arbitrary.